runtime/android/core/BUILD.gn

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/android/rules.gni")
+
+android_library("xwalk_core_java") {
+  deps = [
+    ":xwalk_core_java_resources",
+    "//xwalk/third_party/lzma_sdk:lzma_sdk_java",
+  ]
+  srcjar_deps = [ ":wrapper" ]
```

Comment:
The same thing I mentioned about getting rid of `bridge` in `runtime/android/core_internal/BUILD.gn` also applies here.